aip/general/0193.md

```diff
@@ -107,6 +186,16 @@ enables clients to pull such information programatically--without
 parsing the string error message. For these reasons, `ErrorInfo` is
 required for *every* error response.
 
+The field, `message`, on `Status` is immutable because clients **may**
```

Comment:
Again, I'm not sure the requirement word style for `**may**` is right here since this is a reason for doing something.